themes/default/content/docs/guides/continuous-delivery/gitlab-app.md

```diff
@@ -16,10 +16,16 @@ To enable the integration with your GitLab project, you will need to ensure you
 * [Signup](https://app.pulumi.com/signup) for a Pulumi account with your GitLab identity (or link your GitLab identity with an existing account.)
 * If your GitLab Project is under a GitLab Group, ensure that the group is added to Pulumi as an [organization](https://app.pulumi.com/site/trial)
 
+## Prerequisites
+
+* You must have a GitLab identity associated with your Pulumi account.
+* If you are integrating a GitLab project that is under a group, [add an organization]({{< relref "/docs/intro/console/organizations#creating-an-organization" >}}).
```

Comment:
```suggestion
* If you are integrating a GitLab project that is under a group, [add an organization]({{< relref "/docs/intro/console/organizations#creating-an-organization" >}})
```

Reply:
Looks like the line-break deceived us into thinking this was the end of the line